chat_shell/chat_shell/services/context.py

```
@@ -163,11 +163,44 @@ async def prepare(self) -> ChatContextResult:
+                if self._request.skill_names and "sandbox" in self._request.skill_names:
+                    # Check if sandbox is not already in preload_skills
+                    if "sandbox" not in (self._request.preload_skills or []):
+                        logger.info(
+                            "[CHAT_CONTEXT] Auto-preloading sandbox skill for binary attachment processing"
+                        )
+                        # Add sandbox to preload_skills by modifying the request
+                        if self._request.preload_skills is None:
+                            self._request.preload_skills = []
+                        self._request.preload_skills = list(
+                            self._request.preload_skills
+                        ) + ["sandbox"]
```

Ruff (0.14.13)

Consider `[*list(self._request.preload_skills), "sandbox"]` instead of concatenation

Replace with `[*list(self._request.preload_skills), "sandbox"]`

(RUF005)